From 12981faad54262d0d9eca274941e26bc249e7899 Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Mon, 26 Dec 2022 18:10:30 +0800
Subject: [PATCH] Revert "Linux 5.10.161"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 1a9148dfd8e03835dc7617cee696dd18c0000e99.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 68f8efa0cc30..6f7dae2f1a4e 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 161
+SUBLEVEL = 160
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.3

